src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/PerformCopyRatioSegmentation.java

```diff
@@ -0,0 +1,77 @@
+package org.broadinstitute.hellbender.tools.exome.segmentation;
+
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.ExomeStandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.exome.*;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by davidben on 5/23/16.
+ */
+@CommandLineProgramProperties(
+        summary = "Segment genomic data into regions of constant minor allele fraction.  Only supports one sample input.",
+        oneLineSummary = "Segment genomic data into regions of constant minor allele fraction",
+        programGroup = CopyNumberProgramGroup.class
+)
+public class PerformCopyRatioSegmentation extends CommandLineProgram {
+    protected static final String NUM_STATES_LONG_NAME = "numberOfStates";
+    protected static final String NUM_STATES_SHORT_NAME = "numStates";
+
+
+    @Argument(
+            doc = "Tangent normalized read counts file",
+            shortName = ExomeStandardArgumentDefinitions.TARGET_FILE_SHORT_NAME,
+            fullName =  ExomeStandardArgumentDefinitions.TARGET_FILE_LONG_NAME,
+            optional = false
+    )
+    protected String coverageFile;
+
+
+    @Argument(
+            doc = "Initial number of hidden allele fraction states",
+            fullName = NUM_STATES_LONG_NAME,
+            shortName = NUM_STATES_SHORT_NAME,
+            optional = false
```

Comment:
hmm, i wonder if it might be good to choose a default value here?  perhaps after we do some evaluation on real data, though.